plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-checkout-restricted-coupons.spec.js

```diff
@@ -654,11 +645,16 @@ test.describe( 'Cart & Checkout Restricted Coupons', () => {
 			.getByLabel( 'Email address' )
 			.first()
 			.fill( 'marge@example.com' );
-		await page.getByRole( 'button', { name: 'Place order' } ).click();
+
+		await page
+			.getByRole( 'link', { name: 'Click here to enter your code' } )
+			.click();
+		await page.getByPlaceholder( 'Coupon code' ).fill( 'email-restricted' );
+		await page.getByRole( 'button', { name: 'Apply coupon' } ).click();
 
 		await expect(
 			page.getByText(
-				'Sorry, it seems the coupon "email-restricted" is not yours - it has now been removed from your order.'
+				'Please enter a valid email to use the coupon code "email-restricted".'
```

Comment:
This part only checks the error message in the Checkout block. What are your thoughts on checking the error message in the Cart block and the cart and checkout shortcodes as well?